src/NLog/Layouts/JSON/JsonLayout.cs

```diff
@@ -341,13 +347,18 @@ private void BeginJsonProperty(StringBuilder sb, string propName, bool beginJson
         {
             if (beginJsonMessage)
             {
-                sb.Append(SuppressSpaces ? "{\"" : "{ \"");
+                sb.Append((IndentJson || SuppressSpaces) ? "{" : "{ ");
             }
             else
             {
-                sb.Append(SuppressSpaces ? ",\"" : ", \"");
+                sb.Append((IndentJson || SuppressSpaces) ? "," : ", ");
```

Comment:
Do not think you should check `IdentJson` here also, since `SuppressSpaces` is false by default.